test/images/agnhost/porter/porter.go

```diff
@@ -34,20 +35,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const tcpPrefix = "SERVE_PORT_"
-const sctpPrefix = "SERVE_SCTP_PORT_"
-const tlsPrefix = "SERVE_TLS_PORT_"
+const (
+	portPrefix = "SERVE_PORT_"
+	tcpPrefix  = "SERVE_TCP_PORT_"
+	udpPrefix  = "SERVE_UDP_PORT_"
+	tlsPrefix  = "SERVE_TLS_PORT_"
+	sctpPrefix = "SERVE_SCTP_PORT_"
+)
 
 // CmdPorter is used by agnhost Cobra.
 var CmdPorter = &cobra.Command{
 	Use:   "porter",
 	Short: "Serves requested data on ports specified in ENV variables",
-	Long: `Serves requested data on ports specified in environment variables of the form SERVE_{PORT,TLS_PORT,SCTP_PORT}_[NNNN]. 
+	Long: `Serves requested data on ports specified in environment variables of the form SERVE_{PORT,TCP_PORT,UDP_PORT,TLS_PORT,SCTP_PORT}_[NNNN]. 
 	
 eg:
 * SERVE_PORT_9001 - serve TCP connections on port 9001
-* SERVE_TLS_PORT_9002 - serve TLS-encrypted TCP connections on port 9002
-* SERVE_SCTP_PORT_9003 - serve SCTP connections on port 9003
+* SERVE_TCP_PORT_9002 - serve TCP connections on port 9002
```

Comment:
should probably document `SERVE_PORT_*` and `SERVE_TCP_PORT_*` on the same line and using the same port number, just to make it clearer that it's two ways of saying the same thing, as opposed to being subtly different in some way.